a/apache-nifi/build_info.json

```diff
@@ -2,6 +2,8 @@
     "package_name" : "apache-nifi",
     "github_url":"https://github.com/apache/nifi",
     "version": "rel/nifi-1.17.0",
+    "default_branch": "main",
+    "build_script" : "apache-nifi_1.17.0_ubi_8.5.sh",
```

Comment:
Line 17 is for any tags, related to generic release versions (branches matching `rel/nifi-`). In case we want to run for any branch line 6 script will be used.